templates/zerver/help/change-your-time-zone.md

```diff
@@ -0,0 +1,13 @@
+# Change your time zone
+
+By default, Zulip will detect your time zone based off of your computer's
+settings. However, if you want to have Zulip display times in a
+particular timezone all of the time, you can set the timezone in the Zulip
+settings.
+
+1. Go to the [Time Zone Settings](/#settings/display-settings) by clicking on
```

Comment:
Let's follow the wording from http://localhost:9991/help/change-your-name exactly. (It's in master, but not chat.zulip.org yet).
A next step is probably to turn it into a macro.